src/silo/query_engine/exec_node/table_scan.h

```diff
@@ -81,6 +83,7 @@ class TableScan : public arrow::acero::ExecNode {
 
    arrow::Status StopProducingImpl() override {
```

Comment:
Why is there actually `StopProducingImpl` and `StopProducing` (in the parent class)?